extensions/ql-vscode/src/view/method-modeling/MethodModeling.tsx

```diff
@@ -40,6 +40,7 @@ export type MethodModelingProps = {
   modelingStatus: ModelingStatus;
   method: Method;
   modeledMethod: ModeledMethod | undefined;
+  showMultipleModels?: boolean;
```

Comment:
I needed to have some way to use the `viewState` in the `MethodModelingView`, so this is just to make use of the `viewState` for now. In a follow-up PR, this will be used.